src/index.js

```diff
@@ -0,0 +1,111 @@
+import React, { useRef, useState, useLayoutEffect } from 'react';
+import { makeDecorator } from '@storybook/addons';
+import { addons } from '@storybook/addons';
+import { Resizable } from 're-resizable';
```

Comment:
[question] Is this a third-party dependency? I don't see it in the package.json.

Reply:
Good catch, thanks.